### PR TITLE
bug/url changes for events and webinars - PRESS0-1282

### PR DESCRIPTION
**Pull request by ramyakrishnai**
## Proposed changes
Update "Events and Webinars" from the redirecting to a 404 page. Instead take the end customer to;
https://www.bluehost.com/blog/events/ 

story : https://jira.newfold.com/browse/PRESS0-1282 

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by ramyakrishnai** [comment]
created this PR to run all the tests https://github.com/bluehost/bluehost-wordpress-plugin/pull/1076

**Comment by circlecube** [comment]
I think ideally we leave those utm query parameters in the url.

**Comment by manikantakailasa** [comment]
> I think ideally we leave those utm query parameters in the url.

updated with utm query params

**Review by circlecube — APPROVED** [review]

**Review by wpalani — APPROVED** [review]

File: tests/cypress/integration/help.cy.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ describe( 'Help Page', function () {
 	} );
 
 	it( 'Is Accessible', () => {
-		cy.checkA11y( '.wppbh-app-body' );
+		cy.checkA11y( '.wppbh-app-body' , {timeout : 20000 });
```
[the remark this answers is not in the document]
Comment (manikantakailasa) [reply tests/cypress/integration/help.cy.js]: Test case was failing even after having the id tag in Dom so added delay.

**Pull request merged**